static/app/utils/getKeyCode.ts

```diff
@@ -0,0 +1,79 @@
+/* key maps and utils retrieved from  https://github.com/jaywcjlove/hotkeys */
+
+/** Includes a lot of leftover unused codes for the future in case we want glyphs for them */
```

Comment:
nit, this type of comment typically should be on multiple lines
```suggestion
/**
 * Includes a lot of leftover unused codes for the future in case we
 * want glyphs for them 
 */
```